website_v2/gcloud-powershell-website-v2/www/static/app/mainController.js

```diff
@@ -0,0 +1,19 @@
+var app = angular.module('powershellSite');
+
+app.controller('MainController',
+    function($scope, $route, $routeParams, $location, $http) {
+    $scope.$route = $route;
```

Comment:
I assumed those needed to be there since the angular documentation showed that they were. Removing them didn't affect anything though, so I'll remove them.